packages/auth-construct/src/construct.ts

```diff
@@ -214,6 +218,37 @@ export class AmplifyAuth
     );
   }
 
+  /**
+   * Define bindCustomAttribute to meet requirements of the Cognito API to call the bind method
+   */
+  public bindCustomAttribute = (
```

Comment:
seems like this method should be private. We should test this functionality via constructor inputs. I see there already are some unit tests exercising this; maybe that is sufficient and we don't need the unit tests that are directly calling this method?

Reply:
Updated.